#!/usr/bin/env python
PACKAGE = "kuri_camera"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("jpeg_quality", int_t, 0, "JPEG Compression Quality", 20,  0, 100)

exit(gen.generate(PACKAGE, "kuri_camera", "KuriCamera"))
